x-pack/test/cases_api_integration/common/plugins/cases/server/plugin.ts

```diff
@@ -39,37 +35,9 @@ export class FixturePlugin implements Plugin<void, void, FixtureSetupDeps, Fixtu
     deps.cases.attachmentFramework.registerExternalReference(getExternalReferenceAttachment());
     deps.cases.attachmentFramework.registerPersistableState(getPersistableStateAttachment());
 
-    const router = core.http.createRouter();
```

Comment:
Logic moved to `x-pack/test/cases_api_integration/common/plugins/cases/server/routes.ts`